packages/rancher-pushprox/charts/templates/pushprox-clients-rbac.yaml

```diff
@@ -11,7 +11,12 @@ rules:
   resourceNames:
   - {{ template "pushProxy.client.name" . }}
 {{- if and .Values.clients.https.enabled .Values.clients.https.useServiceAccountCredentials }}
-- nonResourceURLs: ["/metrics"]
+- nonResourceURLs:
+  - "/metrics"
+  - "/metrics/cadvisor"
+  verbs: ["get"]
+- apiGroups: [""]
+  resources: ["nodes/metrics"]
```

Comment:
Ideally, this should be supplied by the user and not hard-coded; not all ServiceMonitors need this permission.